### PR TITLE
Add mute/unmute option

### PR DESCRIPTION
**Pull request by corest**
Example: mute all support channels `python cli.py mute --include "support-.*"`

### CODE REVIEW

**Comment by marians** [comment]
Cool, thanks!

I would call the command `mute`, even though it's a mute/unmute toggle.

Will test later.

**Comment by JosephSalisbury** [comment]
Looking at https://github.com/ErikKalkoken/slackApiDoc/blob/2b0fa0912580500ba351000f16c0837c29fa7b52/users.prefs.get.md, and the field `muted_channels`, I reckon you can grab muted channels from the API

File: README.md [hunk 1]
```diff
@@ -55,6 +55,16 @@ slackutil star --include "cool-.*"
 slackutil unstar --include "uncool-.*"
 ```
 
+### Mutting/unmutting channels
+As there is no API for retrieving current `mute` status, `mute` just sends `/mute` command into channel.
+So if channel was mutted before - it will become unmutted and if it was unmutted - it will become mutted.
```
Comment (JosephSalisbury) [inline README.md]: Random 🇬🇧 drive by - 'muted', not 'mutted'. I think 'mutted' would mean to turn somebody into an ugly dog :D

**Comment by corest** [comment]
Thx Joe, added clear mute/unmute options

File: cli.py [hunk 2]
```diff
@@ -38,12 +39,26 @@ def unstar_channel(channel_id):
     r = requests.get(url, params=params)
     r.raise_for_status()
 
+def get_mutted_channels():
```
Comment (JosephSalisbury) [inline cli.py]: Sorry to be pedantic, but this should be `get_muted_channels`

**Review by JosephSalisbury — APPROVED** [review]
One naming issue for me, but go on as you see fit.

File: cli.py [hunk 3]
```diff
@@ -38,12 +39,26 @@ def unstar_channel(channel_id):
...
+    url = BASE_URL + "users.prefs.get"
+    params = {"token": TOKEN}
+    r = requests.get(url, params=params)
+    r.raise_for_status()
+    data = json.loads(r.text)
+    return data["prefs"]["muted_channels"]
+
+def remute_channel(channel_id):
+    url = BASE_URL + "chat.command"
+    params = {"token": TOKEN, "channel": channel_id, "command": "/mute"}
```
Comment (kopiczko) [inline cli.py:52]: Is it hidden api?

**Review by kopiczko — APPROVED** [review]
I appreciate you and your PR

File: cli.py (hunk 3 above)
Comment (corest) [reply cli.py:52]: Yep, Marian found that here https://github.com/ErikKalkoken/slackApiDoc

File: cli.py [hunk 4]
```diff
@@ -38,12 +39,26 @@ def unstar_channel(channel_id):
     r = requests.get(url, params=params)
     r.raise_for_status()
 
+def get_muted_channels():
+    url = BASE_URL + "users.prefs.get"
+    params = {"token": TOKEN}
+    r = requests.get(url, params=params)
+    r.raise_for_status()
+    data = json.loads(r.text)
+    return data["prefs"]["muted_channels"]
```
Comment (marians) [inline cli.py:48]: If `"muted_channels"` does not exist (e. g. the response format changed), this will throw an exception. Do we want to handle this?

Comment (marians) [reply cli.py:48]: Thinking about it, we also don't handle request exceptions. So let's not bother in this PR.

**Review by marians — APPROVED** [review]
Works like charm!

**Pull request merged**